test.py

```diff
@@ -1,25 +1,6 @@
-from typing import Optional, Union
```

Comment:
Feels like this has crept in in a larger PR, and probably actually needs a `git rm`?

Reply:
I've actually `git rm`'ed it before but it keeps coming back for some reason. It's in the gitignore